auth0/structure_auth0_connection.go

```diff
@@ -358,7 +358,7 @@ func flattenConnectionOptionsSAML(options *management.ConnectionOptionsSAML) (in
 		"non_persistent_attrs":     options.GetNonPersistentAttrs(),
 		"entity_id":                options.GetEntityID(),
 		"metadata_url":             options.GetMetadataURL(),
-		"metadata_xml":             options.GetMetadataXML(),
+		"metadata_xml":             String(d, "options.0.metadata_xml"), // Does not get read back.
```

Comment:
Had to include a couple of fixes for the metadata_xml as the tests were failing when running against the real tenant, they were only passing cuz we were making them pass through manipulating the http recordings.